azure-pipelines.yml

```diff
@@ -29,11 +29,11 @@ stages:
     - script: |
         echo Hello
         # the following is copied from jenkins
-        # set -ex
-        # python2 setup.py bdist_wheel
-        # python3 setup.py bdist_wheel
-        # mkdir -p target/python-wheels/
-        # cp dist/swsssdk-2.0.1-{py2,py3}-none-any.whl target/python-wheels/
+        set -ex
+        pip3 install setuptools
```

Comment:
please use ubuntu-20.04 pool, save sonicbld

also add unit test coverage, and test results analysis.

Reply:
replaced by PR# 105, because I can't reopen this PR.